Alisha/data/lab3.ipynb

```diff
@@ -0,0 +1,402 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #13.    data_subset['Formatted Date'] = pd.to_datetime(data_subset['Formatted Date'], errors='coerce')
</pre><p>add <code>utc=True</code></p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/suparthghimire/data-analysis-and-visualization-bca-2025/pull/6/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='6'/>